src/app/node-data/component.ts

```diff
@@ -103,12 +105,14 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
     private readonly _datacenterService: DatacenterService,
     private readonly _nodeDataService: NodeDataService,
     private readonly _settingsService: SettingsService,
+    private readonly _params: ParamsService,
```

Comment:
```suggestion
    private readonly _paramsService: ParamsService,
```

Reply:
this is how it's being used throughout the app 
<img width="1224" alt="image" src="https://user-images.githubusercontent.com/56511988/181773374-11fc076a-489c-4bea-863f-8b59361f83a1.png">

Reply:
hmm, ideally we should append but either u can skip or change .it